tools/staticbuild/build_wheel.sh

```diff
@@ -18,11 +18,30 @@
 # under the License.
 
 # This script builds the wheel for binary distribution and performs sanity check.
+# To be used only on vanilla Ubuntu-14.04 to ensure backwards compatibility
+# of all dependencies of MXNet.
+
+set -e
+
+echo $(pwd) > pwd.txt
+export SRC="`sed -E 's/(.*incubator\-mxnet)(.*)/\1/g' pwd.txt`"
+cd $SRC
+
+source tools/staticbuild/install_prereq.sh
```

Comment:
just call it instead of sourcing it? as it looks like you want to set variables or something.

Reply:
yes

Reply:
why are you sourcing it?